terraform/main.tf

```diff
@@ -223,8 +223,9 @@ resource "google_cloud_run_v2_service" "pudl-superset" {
       resources {
         limits = {
           cpu    = "4"
-          memory = "2048Mi"
+          memory = "4096Mi"
         }
+        startup_cpu_boost = true
```

Comment:
We'll have to play around with the Cloud Run resources and superset row limit to make everything feel smooth. I decided not to allocate a minimum number of instances because it would cost us about [$200 a month](https://cloud.google.com/products/calculator?hl=en&dl=CjhDaVJqTjJNd1pXSmxOeTAzTWpObExUUmhaREV0WWpZeVpTMW1OV1ptWlRCak9UZGpORGtRQVE9PRAcGiQ5QUREREM5MS1FQjlELTRFRUUtQTIxRC0wQ0E5QUIzNUI4QkI). I figured a memory and start CPU boost was reasonable.